pkg/crc/cluster/monitoring.go

```diff
@@ -20,20 +20,28 @@ func StartMonitoring(ocConfig oc.Config) error {
 		return err
 	}
 
+	indexForClusterMonitoringDeploymentKind := getIndexInOverridesForObjectName(cv, "cluster-monitoring-operator")
+	indexForClusterMonitoringCVOKind := getIndexInOverridesForObjectName(cv, "monitoring")
+
+	if indexForClusterMonitoringDeploymentKind != -1 && indexForClusterMonitoringCVOKind != -1 {
+		_, _, err = ocConfig.RunOcCommand("patch", "clusterversion/version",
+			"--type", "json",
+			"--patch", fmt.Sprintf(`'[{"op":"remove", "path":"/spec/overrides/%d"},{"op":"remove", "path":"/spec/overrides/%d"}]'`,
+				indexForClusterMonitoringDeploymentKind, indexForClusterMonitoringCVOKind-1))
```

Comment:
why `indexForClusterMonitoringCVOKind-1` doesn't `getIndexInOverridesForObjectName()` return the correct index?

Reply:
It does return correct index but as soon as we run json patch the index change since the first patch from the list removed so it is shifted.

Reply:
This is assuming `indexForClusterMonitoringDeploymentKind` is smaller than `indexForClusterMonitoringCVOKind`, if this is not true, you'll be removing a random override, if `indexForClusterMonitoringCVOKind` is 0, this will fail too.
Maybe remove each override independently, ie run `oc` twice?